### PR TITLE
Feature/improve color accessibility across app

### PR DESCRIPTION
**Pull request by oftheheadland**
## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3342761

## Main Changes:
* Switched to darker Red, Green, and Purple colors for the maps, icons, and badges for better contrast and accessibility. 
* Moved colors into a single shared file where we can easily edit them in the future

## Steps To Test:
1. Navigate to http://localhost:3000/community/020700100102/overview
Verify the Permitted Dischargers on the Overview panel are a gray color and distinguishable.
Verify Identified Issues tab waterbodies are a darker red and the dischargers are gray.

2. Navigate to http://localhost:3000/community/salt%20lake%20city/overview
Verify the lines are clearly visible through the areas.

3. Run Accessibility tools on the waterbody report page, check for Badge contrast ratios to be passing:
http://localhost:3000/waterbody-report/MDE_EASP/MD-02140204
http://localhost:3000/waterbody-report/21AWIC/AL03150110-0202-200

4. Navigate to http://localhost:3000/community/030501110109/overview
Verify points are not black dots and are displayed correctly.

5. Check State page. For Water Quality Overview check that Highcharts is using the new colors and they are legible.
Then check that Advanced Search map is using the darker colors.
http://localhost:3000/state/KS/water-quality-overview

### CODE REVIEW

**Comment by coobr01** [comment]
Is that code smell ok @oftheheadland ?

**Comment by oftheheadland** [comment]
@coobr01 yes, we're ignoring a few kinds of code smells that don't really apply to us.
@cschwinderg  gave a good explanation:

"We purposefully used the same variable names in the block scope so we could do setSomeState({ data });, instead of setSomeState({ data: tempData });"

The reason it popped up again after marking it resolved is because the Prettier settings are still wrong and parenthesis were added/removed. I'll fix that after this branch is merged

**Comment by coobr01** [comment]
> @coobr01 yes, we're ignoring a few kinds of code smells that don't really apply to us.
> @cschwinderg gave a good explanation:
> 
> "We purposefully used the same variable names in the block scope so we could do setSomeState({ data });, instead of setSomeState({ data: tempData });"
> 
> The reason it popped up again after marking it resolved is because the Prettier settings are still wrong and parenthesis were added/removed. I'll fix that after this branch is merged

Great!

File: app/client/src/components/pages/Community/components/tabs/IdentifiedIssues.js [hunk 1]
```diff
@@ -166,7 +166,7 @@ function IdentifiedIssues({ esriModules, infoToggleChecked }: Props) {
         },
         symbol: {
           type: 'simple-marker', // autocasts as new SimpleMarkerSymbol()
-          color: '#246007',
+          color: colors.green(),
           style: 'diamond',
         },
```
[the remark this answers is not in the document]
Comment (oftheheadland) [reply app/client/src/components/pages/Community/components/tabs/IdentifiedIssues.js:171]: Will remove the color property of this symbol

File: app/client/src/components/pages/Community/components/tabs/Monitoring.js [hunk 2]
```diff
@@ -264,7 +266,7 @@ function Monitoring({ esriModules, infoToggleChecked }: Props) {
     plotStations(
       Graphic,
       tempDisplayedMonitoringStations,
-      '#c500ff',
+      colors.lightPurple(),
       monitoringStationsLayer,
     );
```
[the remark this answers is not in the document]
Comment (oftheheadland) [reply app/client/src/components/pages/Community/components/tabs/Monitoring.js:271]: I agree with removing the color parameters, simpler is better

File: app/client/src/components/pages/State/lookups/surveyMapping.js [hunk 3]
```diff
@@ -1,3 +1,15 @@
+// Green
+// Old: #8cc63f
+// New: #cb223e
+
+// Red:
+// Old: #f93b5b
+// New: #20800c
```
[the remark this answers is not in the document]
Comment (oftheheadland) [reply app/client/src/components/pages/State/lookups/surveyMapping.js:7]: looks like it thanks for catching!

**Comment by oftheheadland** [comment]
@cschwinderg I applied your suggestions 👍  Good find with Vermont's colors. I'm okay with the new look of it because the "Very Good" green color looks like a mix of the "Good" green and the "Excellent" Blue. I'm open to suggestions

**Comment by coobr01** [comment]
Great peer review and coding you guys, thanks so much!

File: app/client/src/components/pages/Community/components/tabs/IdentifiedIssues.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (cschwinderg) [reply app/client/src/components/pages/Community/components/tabs/IdentifiedIssues.js]: Can you remove the whole symbol object (lines 167 - 170)?

[the remark this answers is not in the document]
Comment (oftheheadland) [reply app/client/src/components/pages/Community/components/tabs/IdentifiedIssues.js]: Done, thanks

**Pull request merged**